GeneratorInterface/Herwig7Interface/test/Herwig7Matchbox_90X_ppToee.py

```diff
@@ -0,0 +1,186 @@
+# Auto generated configuration file
```

Comment:
this config is already obsolete. Please maintain it as a cmsDriver command instead
(same is true for the others)